lightly/api/api_workflow_selection.py

```diff
@@ -82,6 +82,13 @@ def selection(
 
         """
 
+        warnings.warn(
```

Comment:
If you want, you can already remove the selection() method in this PR, otherwise, it is telling to use another deprecated method.

Reply:
Added this to #1251. Will get that merged first.